.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Feature request & feedback
+    url: https://github.com/meilisearch/product/discussions/categories/feedback-feature-proposal
```

We should redirect to the specialized label, no?
```suggestion
    url: https://github.com/meilisearch/product/discussions?discussions_q=label%3Aproduct%3Acore%3Atokenizer
```